group-coordinator/group-coordinator-api/src/main/java/org/apache/kafka/coordinator/group/api/assignor/GroupSpec.java

```diff
@@ -45,18 +45,18 @@ public interface GroupSpec {
     /**
      * Gets the member subscription specification for a member.
      *
-     * @param memberId          The member Id.
+     * @param memberId The member Id.
      * @return The member's subscription metadata.
      * @throws IllegalArgumentException If the member Id isn't found.
      */
-    MemberSubscriptionSpec memberSubscription(String memberId);
```

Comment:
nit: why did we rename it to MemberSubscription?

Reply:
Now that we have `MemberAssignment`, `MemberSubscriptionSpec` looked weird. So I renamed it to align them.